scripts/pg2sphinx_trigger.py

```diff
@@ -1,25 +1,26 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 #
-# read sphinx config file and update all indexes related to input database or input table or index pattern
+# read sphinx config file and update all indexes related to input database or
+# input table or index pattern
 
 import os
-import sys
-import time
-import getpass
 import re
 import sqlite3
-import psycopg2
-import optparse
 import subprocess
-from optparse import OptionParser
+import sys
+from optparse import OptionParser  # pylint: disable=deprecated-module
 
+import psycopg2  # pylint: disable=import-error
 
-def pg_get_tables(sql_query,sql_db):
+
+def pg_get_tables(sql_query, sql_db):
     # add 'EXPLAIN VERBOSE ' to sql string
     sql_query = 'EXPLAIN VERBOSE ' + sql_query
 
-    conn_string = "host='%s' dbname='%s' user='%s' password='%s'" % (CONN_HOST,sql_db,CONN_USER,CONN_PWD)
+    conn_string = "host='%s' dbname='%s' user='%s' password='%s'" % (
+        CONN_HOST, sql_db, CONN_USER, CONN_PWD
+    )
```

Comment:
Might be a good idea to use f string here, more readable and also more modern as the old style string formating.

Reply:
Yes, and also would avoid rising `consider-using-f-string` with current pylint versions: https://pylint.pycqa.org/en/latest/whatsnew/2.11.html#new-checkers

Reply:
i have migrated the project to python3.9.9 and im using fstrings now